cdap-docs/admin-manual/source/installation/cdap-site.rst

```diff
@@ -1,410 +1,43 @@
-       installation of CDAP
-   * - ``security.server.extended.token.expiration.ms``
-     - ``604800000``
-     - Admin tool access token expiration time in milliseconds (defaults to 1 week) (internal)
-   * - ``security.server.maxthreads``
-     - ``100``
-     - Maximum number of threads that the CDAP Authentication Server should use for
-       handling HTTP requests
-   * - ``security.server.token.expiration.ms``
-     - ``86400000``
-     - Access token expiration time in milliseconds (defaults to 24 hours)
-   * - ``security.token.digest.algorithm``
-     - ``HmacSHA256``
-     -  Algorithm used for generating MAC of access tokens
-   * - ``security.token.digest.key.expiration.ms``
-     - ``3600000``
-     - Time duration (in milliseconds) after which an active secret key
-       used for signing tokens should be retired
-   * - ``security.token.digest.keylength``
-     - ``128``
-     - Key length used in generating the secret keys for generating MAC of access tokens
-   * - ``security.token.distributed.parent.znode``
-     - ``/${root.namespace}/security/auth``
-     - Parent node in ZooKeeper used for secret key distribution in distributed mode
-   * - ``ssl.enabled``
-     - ``false``
-     - True to enable SSL
-   * - ``stream.flume.port``
-     - ``10004``
-     -
-   * - ``stream.flume.threads``
-     - ``20``
-     -
-   * - ``stream.notification.threshold``
-     - ``1024``
-     - Size of data, in MB, to be ingested by a stream before a notification is published
-   * - ``thrift.max.read.buffer``
-     - ``16777216``
-     - Maximum read buffer size in bytes used by the Thrift server [`Note 2`_]
-   * - ``twill.java.reserved.memory.mb``
-     - ``250``
-     - Reserved non-heap memory in MB for Twill container
-   * - ``twill.jvm.gc.opts``
-     - | ``-verbose:gc``
-       | ``-Xloggc:&lt;LOG_DIR&gt;/gc.log``
-       | ``-XX:+PrintGCDetails``
-       | ``-XX:+PrintGCTimeStamps``
-       | ``-XX:+UseGCLogFileRotation``
-       | ``-XX:NumberOfGCLogFiles=10``
-       | ``-XX:GCLogFileSize=1M``
-     - Java garbage collection options for all Twill containers; ``<LOG_DIR>`` is the location
-       of the log directory on each machine; note that the special characters are replaced with
-       entity equivalents so they can be included in the XML
-   * - ``twill.no.container.timeout``
-     - ``120000``
-     - Amount of time in milliseconds to wait for at least one container for Twill runnable
-   * - ``twill.zookeeper.namespace``
-     - ``/twill``
-     - Twill Zookeeper namespace prefix
-   * - ``yarn.user``
-     - ``yarn``
-     - User name for running applications in YARN
-   * - ``zookeeper.quorum``
-     - ``127.0.0.1:2181/${root.namespace}``
-     - Zookeeper address host:port
-   * - ``zookeeper.session.timeout.millis``
-     - ``40000``
-     - Zookeeper session time out in milliseconds
+.. include:: ../../build/_includes/cdap-default-table.rst
 
 .. _note 1:
```

Comment:
I think we should remove these two notes and pull their contents into the properties that they are about.